app/src/main/java/vn/mbm/phimp/me/MapImageActivity.java

```diff
@@ -0,0 +1,51 @@
+package vn.mbm.phimp.me;
+
```

Comment:
@Abhi2424shek Yeah I know, check out this 
Uri imageUri = Uri.fromFile(file);
Glide.with(this).load(imageUri).into(imageView);

in new activity.

 Taking the image from map fragment was creating the problem, so made this separate activity to use glide.